### ISSUE TITLE
EPIC 201623069 (Thing 1)

### ISSUE THREAD

**Issue by George-Hookway** [George-Hookway/PBjammy#88]
![201623069 - 8 - SG - Echelle](https://github.com/user-attachments/assets/53994ee5-36b8-426e-92a0-335723474151)
![201623069 - 8 - SG - Mode ID](https://github.com/user-attachments/assets/f831691f-8a8d-44d2-b5a5-a404cc6eaac2)
![201623069 - 8 - SG - Peakbagged Echelle](https://github.com/user-attachments/assets/e48ac818-bda8-409f-b3fa-9476b771f5ce)
![201623069 - 8 - SG - Peakbagging](https://github.com/user-attachments/assets/1c205cd6-d3b5-44d1-8067-de5f2b3f2d91)

@nielsenmb @grd349

**Comment by nielsenmb** [comment]
ng=1 is in the wrong location for this one :(

It should be at ~850muHz in my opinion.

**Comment by George-Hookway** [comment]
Yeah, I think you're right. I'm currently re-running it with a lower Np, which might fix it like with Lightning McQueen
